src/ui/Misc/TextEllipsisTooltip.ts

```diff
@@ -0,0 +1,47 @@
+import 'styling/_TextEllipsisTooltip';
```

Comment:
Always hard to know where to put those kind of class, but I think it should belong the the `/src/utils` folder.

I saw what it looks like and it is very nice looking and interesting for end user, I think.

But to play devil's advocate, would a simple "title" attribute on the element not be enough ? 
It is not as nice looking but : 

* It's one less thing we would have to maintain.
* It is well handled by all type of devices (mobile included), which I don't know how this component handle currently.
* It is more "standard" across all other components. If we want to use this widget for the ML facet, we'd want to use it across all components that needs to display ellipsed/truncated text, otherwise it's going to be inconsistent.
* Helps for accessibility.

Reply:
Ok I will move it.

Going against the devil
- A title is really not visually as nice or obvious, and it takes a little while to show up. (and we can put both)
- For mobile testing with chrome if you press on the title it shows up, for the facet value mobile is not working as well (maybe using a title too would help)
- If UX recommends it we could use it elsewhere too
- It's already accessible

We can have a chat with UX to see if the title attribute avenue is sufficient

Reply:
> For mobile testing with chrome if you hold press on the text, the tooltip shows up

Are you sure that is the correct behaviour of a long press on all devices ? 
Normally this would open context menu on IOS, for example.